docs/02-guides/server-side-events/create-an-integration.mdx

```diff
@@ -7,7 +7,135 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+This guide will help you to create an integration that will dispatch events to external services.
+
+We will create a new integration to dispatch events to [Wehook.cool](https://webhook.cool/).
+
+:::info
+
+Front-Commerce already provide a Webhook integration. This guide will show you how to create an integration from scratch. And will explain you the basics of integrations so you can write your own
+
+:::
+
+## Create an integration file
+
+First you need to create an integration file let's say we will call our integration Coolwebhook
+
+Let's create a new file in `app/server-evets/integrations/CoolWebhookIntegration.ts`
+
+Integrations are classes that extends `Integration interface`, let's make a simple copy paste of this code:
+
+```ts title="app/server-evets/integrations/CoolWebhookIntegration.ts"
+import type { Integration } from "@front-commerce/core/server-events/integrations";
+
+export default class CoolWehbookIntegration implements Integration {
+  compatibleEvents: string[] | "*" = "*";
+
+  webhookId: string;
+
+  constructor(webhookId: string) {
+    this.webhookId = webhookId;
+  }
+
+  handle(messageId: string, payload: any) {
+    console.log(messageId);
+    console.log(payload)
+  }
+}
+```
+
+In this class, we have four entities :
+- `compatibleEvents`: this indicates which events the integration will handle, it can be `*` to handle all events or an array of string that contains compatible events.
+- `webhookId`: this is the id of WebhookCool.
```

Comment:
Same as https://github.com/front-commerce/developers.front-commerce.com/pull/843/files#r1483132197, I don't think we should document basic javascript, this is not extending the original interface in any way, so there's no need to mention it. The only things to mention here is the [Integration interface](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/core/server-events/types/integration.ts#L1-4)